Fixed an issue which caused failures when determining the path to ``setup.py`` files.
